apps/system/js/app_window.js

```diff
@@ -223,6 +223,12 @@
     this.getTopMostWindow()._setActive(enable);
   };
 
+  AppWindow.prototype.inScope = function(scope) {
+    var url = new URL(this.config.url);
+    var path = url.hostname + url.pathname;
+    return path.indexOf(scope) === 0;
```

Comment:
Maybe this should only match browser windows as being in scope? Currently tapping a bookmark at m.facebook.com will re-use a window of the Facebook app, I'm not sure we want to go that far just yet, I think we need to treat apps separately.